### PR TITLE
Enable IPv6 on cdi http file host.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Enable ipv6 for the cdi file hosts in case of an ipv6 only cluster for testing. The registry file host is already ipv6 enabled from what I could tell.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tools/cdi-func-test-file-host-init/nginx.conf [hunk 1]
```diff
@@ -34,6 +35,7 @@ http {
         server_name localhost;
 
         listen 81;
+        listen [::]:82;
```
Comment (mhenriks) [inline tools/cdi-func-test-file-host-init/nginx.conf]: should this be `[::]:82`?

Comment (awels) [reply tools/cdi-func-test-file-host-init/nginx.conf]: Yes... D'oh!

**Comment by awels** [comment]
Not sure why https://github.com/kubevirt/containerized-data-importer/pull/1130 was closed, but this PR is the same.

**Pull request merged**